packages/form-js-viewer/test/spec/Form.spec.js

```diff
@@ -638,21 +695,32 @@ describe('Form', function() {
     // then
     expect(submission.data).to.eql({
       creditor: 'Jane Doe Company',
-      amount: 456,
       invoiceNumber: 'C-123',
+      amount: 456,
       approved: true,
-      approvedBy: 'John Doe'
+      approvedBy: 'John Doe',
+      product: null,
+      language: null
     });
 
-    expect(submission.errors).to.eql({});
+    expect(submission.errors).to.be.empty;
```

Comment:
Does this work on objects?

Reply:
Yes: `When the target is a non-function object, .empty asserts that the target doesn’t have any own enumerable properties`

Reply:
:+1: